src/app/app.component.scss

```diff
@@ -14,3 +14,12 @@
 	border-top: 1px solid $color-pf-blue-300;
 	margin-top: 2%;
 }
+
+.app {
+	padding: em(96) 0 em(66) 0;
```

Comment:
when the user is no logged in, padding-bottom should be 0. Currently we are using  .containerPadd-nologin{ padding-bottom: 0; }

As we have moved the padding to outer element , we need padding-bottom:0 in that scenario for same element